nixos/modules/services/matrix/draupnir.md

```diff
@@ -0,0 +1,80 @@
+# Draupnir (Matrix Moderation Tool) {#module-services-draupnir}
```

Comment:
@teutat3s (sorry for replying this way, github only allows threading in diff comments...)
In reply to [your comment about tests](https://github.com/NixOS/nixpkgs/pull/274052#issuecomment-2155708655)

I'm aware that draupnir doesnt officially support pantalaimon (see module warns), thanks for reminding me though, given i'd completely forgotten (and tbh just copied the tests for mjolnir)

I wonder if i should drop the tests *with* pantalaimon? or well, at least rework it to not use pantal...

Reply:
People coming from mjolnir will probably have pantalaimon running. It would be good to notice when things break in obvious ways.

Reply:
The pantalaimon/e2ee warning at eval time is enough, is it not?